### PR TITLE
Adding config for central-settlement to send messages to the Notifica…

### PR DESCRIPTION
**Pull request by elnyry**
…tions (topic)

### CODE REVIEW

File: centralsettlement/configs/default.json [hunk 1]
```diff
@@ -7,5 +7,42 @@
...
+                "options": {
+                  "messageCharset": "utf8"
+                },
+                "rdkafkaConf": {
+                  "metadata.broker.list": "{{ (default .Values.config.kafka_host $kafkaHost) }}:{{ .Values.config.kafka_port }}",
+                  "client.id": "cs-prod-notification-event",
+                  "event_cb": true,
+                  "socket.keepalive.enable": true,
+                  "queue.buffering.max.messages": 10000000,
+                  "dr_cb": true
```
Comment (rmothilal) [inline centralsettlement/configs/default.json]: dr_cb should false

File: centralsettlement/configs/default.json [hunk 2]
```diff
@@ -7,5 +7,42 @@
     "HUB_PARTICIPANT": {
         "ID": 1,
         "NAME": "Hub"
-    }
+    },
+    "SIDECAR": {
```
Comment (rmothilal) [inline centralsettlement/configs/default.json]: I don’t believe you need sidecar config here

File: centralsettlement/configs/default.json (hunk 1 above)
Comment (elnyry) [reply centralsettlement/configs/default.json]: Fixed

File: centralsettlement/configs/default.json (hunk 2 above)
Comment (elnyry) [reply centralsettlement/configs/default.json]: Fixed

**Pull request merged**